### PR TITLE
Added info what are the new semantic attributes to be used

### PR DESCRIPTION
**Pull request by marcingrzejszczak**
Related change - https://github.com/open-telemetry/opentelemetry-specification/commit/0b45213858e23aedb6f6de5fab17d2843a9f24c5

### CODE REVIEW

**Review by mateuszrzeszutek — APPROVED** [review]

File: buildscripts/semantic-convention/templates/SemanticAttributes.java.j2 [hunk 1]
```diff
@@ -166,13 +166,13 @@ public final class {{class}} {
   public static final AttributeKey<String> HTTP_HOST = stringKey("http.host");
 
   /**
-   * @deprecated This item has been removed as of 1.13.0 of the semantic conventions.
+   * @deprecated This item has been removed as of 1.13.0 of the semantic conventions. Please use {@link SemanticAttributes.NET_SOCK_PEER_ADDR} instead.
```
Comment (jack-berg) [inline buildscripts/semantic-convention/templates/SemanticAttributes.java.j2:169]: @jkwatson I briefly brought up the question of how long we retain deprecated semantic attributes [here](https://github.com/open-telemetry/opentelemetry-java/pull/4766#discussion_r977606900). If we go with one release cycle, then all of these will be deleted and this change is unnecessary.

**Comment by jkwatson** [comment]
@marcingrzejszczak are you able to run the code generator as well, to get the actual java files up to date?

File: buildscripts/semantic-convention/templates/SemanticAttributes.java.j2 (hunk 1 above)
Comment (mateuszrzeszutek) [reply buildscripts/semantic-convention/templates/SemanticAttributes.java.j2:169]: > Is there any harm in keeping these around longer?

AFAIK no, not really. I think we could keep them for a bit longer if it makes it easier for people to migrate off of them; we're certainly no longer using these in the instrumentation repo.

Comment (jack-berg) [reply buildscripts/semantic-convention/templates/SemanticAttributes.java.j2]: Two versions? Three versions?

Comment (jkwatson) [reply buildscripts/semantic-convention/templates/SemanticAttributes.java.j2]: no idea. Let's raise it as a topic at the Thursday meeting.

Comment (trask) [reply buildscripts/semantic-convention/templates/SemanticAttributes.java.j2]: added to agenda

**Review by jack-berg — APPROVED** [review]

**Pull request merged**